docs/static/configuration.asciidoc

```diff
@@ -510,18 +510,45 @@ doesn't exist versus a field that's simply false. The expression `if [foo]` retu
 
 For more complex examples, see <<using-conditionals, Using Conditionals>>.
 
-[float]
+[discrete]
```

Comment:
I like the content, but I think positionally, we might want to put this under the `@metadata` section - the workaround uses the `@metadata` field, and the context could be useful. Also, I suspect that this topic is _probably_  less relevant to more users than the `@metadata` topic